.storybook/webpack.config.js

```diff
@@ -1,13 +1,10 @@
 const path = require('path');
 
-module.exports = {
-    module: {
-        rules: [
-            {
-                test: /\.css?$/,
-                use: ['style-loader', 'raw-loader'],
-                include: path.resolve(__dirname, '../')
-            }
-        ]
-    }
+module.exports = ({ config, mode }) => {
+    return config;
```

Comment:
should this be here?

Reply:
using default config. I don't think we need to extend it any longer